.github/workflows/test.yml

```diff
@@ -40,3 +40,40 @@ jobs:
           restore-keys: ${{ runner.os }}-jdk${{ matrix.java }}-
       - name: Test
         run: mvn test
+
+  test_graalvm:
+    name: test ${{ matrix.os }} ${{ matrix.java }} GraalVM native-image
+    strategy:
+      fail-fast: false
```

Comment:
I need to remove this, I disabled fail fast during tests but it should be activated.